integration-tests/typings/embrace.d.ts

```diff
@@ -27,17 +27,64 @@ interface EmbraceSpanData {
   attributes: EmbraceSpanAttribute[];
 }
 
+interface EmbraceLogData {
+  trace_id: string;
+  severity_number: number;
+  severity_text: string;
+  body: string;
+  time_unix_nano: number;
+  status: "Unset" | "Ok" | "Error";
+  events: EmbraceSpanEvent[];
+  attributes: EmbraceSpanAttribute[];
+}
+
+interface ParsedLogPayload {
+  resource?: EmbracePayloadResource;
+  metadata?: EmbracePayloadMetadata;
+  info: EmbraceLogData[];
+  warn: EmbraceLogData[];
+  error: EmbraceLogData[];
+}
+interface EmbracePayloadLogs {
```

Comment:
should follow the pattern for spans, this could just have `logs: EmbraceLogData[]` as a property and then below update `EmbracePayload` to have type: "spans" | "logs" and data: EmbracePayloadSpans | EmbracePayloadLogs